src/Microsoft.OData.Core/ODataLibraryCompatibility.cs

```diff
@@ -4,29 +4,39 @@
 // </copyright>
 //---------------------------------------------------------------------
 
-using System.Diagnostics.CodeAnalysis;
+using System;
 
 namespace Microsoft.OData
 {
     /// <summary>
     /// Library compatibility levels.
     /// </summary>
-    [SuppressMessage("Microsoft.Design", "CA1008:EnumsShouldHaveZeroValue")]
-    public enum ODataLibraryCompatibility
+    [Flags]
+    public enum ODataLibraryCompatibility : long
     {
+        /// <summary>
+        /// No specific version or feature set.
+        /// </summary>
+        None = 0, // No flags set
+
+        /// <summary>
+        /// When enabled, the CSDL Scale and SRID "variable" value will be written "Variable" instead of "variable" for compatibility with older versions of the library.
+        /// </summary>
+        UseLegacyVariableCasing = 1 << 0,
+
         /// <summary>
         /// Version 6.x
         /// </summary>
-        Version6 = 060000,
+        Version6 = 1 << 6,

```

Comment:
Hmm, why `1 << 6`?, `1 << 6` is `1000000` in binary (64 in decimal). Does this assume that we'll have no more than 6 enum flags?

Reply:
I believe these are just enum flags.. I don't think they have anything to do with how many enum flags we'll have or not have.